hooks/usePayment.js

```diff
@@ -0,0 +1,30 @@
+import { useState } from 'react';
+import { loadStripe } from '@stripe/stripe-js';
+
+const stripePromise = loadStripe(process.env.STRIPE_PUBLIC_KEY);
+
+const getPaymentURL = (e = 1, n) =>
+  `/api/payment?e=${Number(e)}${n ? `&n=${Number(n)}` : ''}`;
```

Comment:
Personal preference: I find this more readable
```js
'/api/payment?e='
  + Number(e)
  + n ? '&n=' + Number(n) : '';
```

Reply:
Changed it to make it use a more proper API (`URLSearchParams`).